docs/CHANGELOG.md

```diff
@@ -1,9 +1,61 @@
+### v3.0.0 (unreleased)
+
+**Breaking changes:**
+
+- Update GraphQL requirement to version 15, which supports a new `extensions` property where join-monster config lives. The config keys and values are unchanged, but they must be nested under an `extensions: { joinMonster: ... }}` property on the GraphQLObjectTypes and fields using join-monster. To upgrade, you must move any non-standard keys off of your `GraphQLObjectType`s or field configs into the `extensions` of the same field. So, something like this:
```

Comment:
not all unchanged—the two jm* ones are different

Reply:
Ah right, will update.